src/components/Header/Tools.tsx

```
@@ -15,7 +15,7 @@ import UserToggle from './UserToggle';
 import ToolbarToggle from './ToolbarToggle';
 import SettingsToggle, { SettingsToggleDropdownGroup } from './SettingsToggle';
 import cookie from 'js-cookie';
-import { ITLess, getSection } from '../../utils/common';
+import { ITLess, getSection, LIGHTWELL_PATH } from '../../utils/common';
```

ESLint

[error] Member 'LIGHTWELL_PATH' of the import declaration should be sorted alphabetically.

(sort-imports)